core/templates/dev/head/pages/collection_player/CollectionPlayer.js

```diff
@@ -283,6 +283,15 @@ oppia.controller('CollectionPlayer', [
       $location.hash(id);
       $anchorScroll();
     };
+
+    $scope.closeOnClickingOutside = function() {
+      $scope.explorationCardIsShown = false;
+    };
+
+    $scope.onClickStopPropogation = function($event) {
```

Comment:
Propagation is misspelled. Also, call the argument `evt`.

Reply:
Done